layouts/shortcodes/keps-data.html

```diff
@@ -0,0 +1,81 @@
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.css">
+
+<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.js"></script>
+<script type="text/javascript">
+  $(document).ready( function () {
+    $('#keps-table').DataTable({
+      lengthMenu: [
+        [10, 25, 50, -1],
+        [10, 25, 50, 'All'],
+      ],
+    });
+});
+</script>
+
+<div>
+  <table id="keps-table" class="table table-hover mt-2 col-md-12 table-sm">
+    <thead class="thead-light">
+      <tr>
+        <th scope="col" class="col-md-1">KEP Number</th>
+        <th scope="col" class="col-md-3">Title</th>
+        <th scope="col" class="col-md-2">SIG</th>
+        <th scope="col" class="col-md-2">Author</th>
+        <th scope="col" class="col-md-1">Created</th>
+        <th scope="col" class="col-md-1">Last Updated</th>
+        <th scope="col" class="col-md-1">Milestone</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{ $keps := getJSON "https://storage.googleapis.com/k8s-keps/keps.json" }}
+      {{ range $index, $data := $keps }}
+          <tr>
+            <td>
+              <a href="https://features.k8s.io/{{ $data.kepNumber }}">
+                {{ printf "%s" $data.kepNumber }}
+              </a>
+            </td>
+            <td>
+              <a href="https://github.com/kubernetes/enhancements/tree/master/keps/{{$data.owningSig}}/{{$data.name}}#summary">
+                {{ printf "%s" $data.title }}
+              </a>
+            </td>
+            <td>
+              <a href=" https://github.com/kubernetes/community/blob/master/{{ $data.owningSig }}/README.md">
```

Comment:
Might be a good idea to make branch names and/or repo URLs `Site` variables (in `config.toml`) so when `master` -> `main` happens or if the repo is renamed, we have to switch it in one place only.

Reply:
We could define these repo and branch names at the top of the shortcode (that's what I'd do).